FSJ_django20_project/FSJ/templates/FSJ/award.html

```diff
@@ -0,0 +1,35 @@
+{% extends "FSJ/base_with_banner.html" %}
+{% load i18n %} {# NOTE YOU MUST LOAD i18n to every HTML page even though it's also loaded to base.html #}
+
+{% block heading %}
+    {# Translators: Add award page heading #}
+    {% trans "Add Award" %}
+{% endblock %}
+
+{% block content %}
+    <div>
+        {# Translators: Greeting to the user #}        
```

Comment:
Greeting to the user?

Reply:
This was leftover from when I was testing the page, I'll remove it